src/js/cards.js

```diff
@@ -26,6 +48,10 @@ export const DEFAULT_ATTACK_RATE = 3000;
 
    A typical card that doesn't stay in play when used will have at least: 
      cost, permanent=false, target, targetCount, description
+
+   For imageName:
```

Comment:
Why do you need an `imageName` key? Why not define an object with the card names as keys and the images as values and look the cards up by name?